test/e2e/apimachinery/namespace.go

```diff
@@ -481,7 +480,7 @@ var _ = SIGDescribe("OrderedNamespaceDeletion", func() {
 	f := framework.NewDefaultFramework("namespacedeletion")
 	f.NamespacePodSecurityLevel = admissionapi.LevelBaseline
 
-	f.It("namespace deletion should delete pod first", feature.OrderedNamespaceDeletion, framework.WithFeatureGate(features.OrderedNamespaceDeletion), func(ctx context.Context) {
```

Comment:
We'll keep the `WithFeatureGate` because it provides information but does not disable running the test in most CI (unless the feature were off-by-default)

but we should drop the e2e feature

https://groups.google.com/a/kubernetes.io/g/dev/c/uVga86nJDuI

Reply:
Thank you for the review. Updated